src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,56 @@
+import { ReactPlugin, withAITracking } from '@microsoft/applicationinsights-react-js';
+import { ApplicationAnalytics, ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ComponentType } from 'react';
+import ITelemetry from './ITelemetry';
+
+class Telemetry implements ITelemetry {
+  private appInsights: ApplicationInsights;
+  private config: any;
+
+  constructor() {
+    this.config = {
+      instrumentationKey: process.env.REACT_APP_INSTRUMENTATION_KEY,
```

Comment:
What would happen if the instrumentation key is not provided?

Reply:
It won't send events to the AzureMonitor instance